CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 # Change Log
 
 ## 0.9.10
+- Added indentation style rules to the lint module
```

Comment:
should probably be "indentation-style", and contain a reference to the file explaining the rules